dali/test/python/test_operator_readers_nemo_asr.py

```diff
@@ -206,30 +222,30 @@ def test_decoded_vs_generated(batch_size=3):
 def nemo_pipe(nemo_op, path, read_text, read_sample_rate, dtype, downmix):
     if read_sample_rate:
         audio, sr = nemo_op(manifest_filepaths=path, read_sample_rate=read_sample_rate,
-                read_text=read_text, dtype=dtype, downmix=downmix)
+                            read_text=read_text, dtype=dtype, downmix=downmix)
         return audio, sr
     elif read_text:
         audio, text = nemo_op(manifest_filepaths=path, read_sample_rate=read_sample_rate,
-                read_text=read_text, dtype=dtype, downmix=downmix)
+                              read_text=read_text, dtype=dtype, downmix=downmix)
         return audio, text
     else:
         audio = nemo_op(manifest_filepaths=path, read_sample_rate=read_sample_rate,
-                read_text=read_text, dtype=dtype, downmix=downmix)
+                        read_text=read_text, dtype=dtype, downmix=downmix)
         return audio
 
 
 def test_nemo_asr_reader_alias():
     for read_sr, read_text in [(True, False), (False, True), (False, False)]:
         for dtype in [types.INT16, types.FLOAT]:
             for downmix in [True, False]:
-                new_pipe = nemo_pipe(fn.readers.nemo_asr, [nemo_asr_manifest], read_sr, read_text, dtype, downmix)
-                legacy_pipe = nemo_pipe(fn.nemo_asr_reader, [nemo_asr_manifest], read_sr, read_text, dtype, downmix)
+                new_pipe = nemo_pipe(fn.readers.nemo_asr, [
+                                     nemo_asr_manifest], read_sr, read_text, dtype, downmix)
+                legacy_pipe = nemo_pipe(fn.nemo_asr_reader, [
+                                        nemo_asr_manifest], read_sr, read_text, dtype, downmix)
```

Comment:
This line break looks strange to happen in the list opening. Maybe it would be better to break this differenty, e.g. like:
```python
            for downmix in [True, False]:
                new_pipe = nemo_pipe(fn.readers.nemo_asr, [nemo_asr_manifest], read_sr, read_text,
                                     dtype, downmix)
                legacy_pipe = nemo_pipe(fn.nemo_asr_reader, [nemo_asr_manifest], read_sr, read_text,
                                        dtype, downmix)
```